src/utils.js

```diff
@@ -0,0 +1,4 @@
+export function isObject(value) {
+  var type = typeof value;
+  return !!value && (type === 'object' || type === 'function');
```

Comment:
Naw that's straight up Lodash's implementation. It's [straight copypasta](https://github.com/lodash/lodash/blob/4.6.1-npm/isObject.js).